railties/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Created rake restart task. 
```

Comment:
It would be nice with an extra sentence that says something like: "Restarts your Rails app by touching the "tmp/restart.txt" file :smile: